Werewolf for Telegram/Werewolf Node/Werewolf.cs

```diff
@@ -761,8 +761,11 @@ public void HandleReply(CallbackQuery query)
 
                 if (player.CurrentQuestion.QType == QuestionType.Lynch)
                 {
-                    var msg = GetLocaleString("PlayerVotedLynch", player.GetName(), target.GetName());
-                    SendWithQueue(msg);
+                    if (DbGroup.EnableSecretLynch == false)
+                    {
+                        var msg = GetLocaleString("PlayerVotedLynch", player.GetName(), target.GetName());
```

Comment:
Rather than not showing anything at all, we should provide them with a message saying how many have voted out of how many alive, which updates periodically.  I can add this later though